code/game/objects/items/weapons/storage/toolbox.dm

```diff
@@ -82,7 +83,7 @@
 	new /obj/item/weapon/crowbar/red(src)
 	new /obj/item/weapon/wirecutters(src, "red")
 	new /obj/item/device/multitool(src)
-	new /obj/item/clothing/gloves/color/red/insulated(src)

```

Comment:
Syndicate's corporate colors are black and red. Combat gloves are black, and they look better even after sprite changes.